utility/pom.xml

```diff
@@ -58,6 +58,26 @@
 
   <build>
     <plugins>
+        <groupId>org.vafer</groupId>
```

Comment:
`<plugin>` is missing here, causing:

```
[FATAL] Non-parseable POM utility/pom.xml: Unrecognised tag: 'groupId' (position: START_TAG seen ...<plugins>\n        <groupId>... @61:18)  @ line 61, column 18
```

Reply:
My apologies for late reply.  Sure, I will make the necessary changes and give a single pull request.  Thank you very much for suggestions.